functable.c

```diff
@@ -82,6 +82,7 @@ static void init_functable(void) {
         ft.longest_match = &longest_match_sse2;
         ft.longest_match_slow = &longest_match_slow_sse2;
 #  endif
+        ft.crc32 = &crc32_chorba_sse;
```

Comment:
```c
#ifdef X86_CHORBA_SSE_CRC
```
Because the function is defined only if macro ``X86_CHORBA_SSE_CRC`` is defined.